src/table/utils/handle-accessibility.js

```diff
@@ -38,8 +38,20 @@ export const handleClickToFocusBody = (cell, rootElement, setFocusedCellCoord, k
   removeAndFocus([rawRowIdx + 1, rawColIdx], rootElement, setFocusedCellCoord, keyboard);
 };
 
-export const handleClickToFocusHead = (columnIndex, rootElement, setFocusedCellCoord, keyboard) => {
-  removeAndFocus([0, columnIndex], rootElement, setFocusedCellCoord, keyboard);
+export const handleClickToFocusCell = (
```

Comment:
hmm sorry if I was unclear, it is the body `handleClickToFocusBody` that we want to rewrite. But I just realized `removeAndFocus` does exactly what you want. If you calculate the `newCoord` inside the component you can use it as is

Reply:
Yes removeAndFocus is the one has been used in all of them. I'll refactor this one by rows.length in Total component.